applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_stationary_embedded_solver.py

```diff
@@ -38,26 +44,77 @@ def AddVariables(self):
         # Add distance variable to represent the embedded skin
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISTANCE)
 
-
     def Initialize(self):
 
-        # Avoid level set zeros using positive epsilon
-        tol = 1.0e-12
-        for node in self.GetComputingModelPart().Nodes:
-            dist = node.GetSolutionStepValue(KratosMultiphysics.DISTANCE)
-            if abs(dist) < tol:
-                node.SetSolutionStepValue(KratosMultiphysics.DISTANCE, 0, tol)
-
-        # Deactivate elements in negative distance region
-        for element in self.GetComputingModelPart().Elements:
-            n_pos = 0
-            for node in element.GetGeometry():
-                if node.GetSolutionStepValue(KratosMultiphysics.DISTANCE) > 0.0:
-                    n_pos += 1
-            if n_pos == 0:
-                element.Set(KratosMultiphysics.ACTIVE, False)
+        if self.settings["use_mls_constraints"].GetBool():
+            if self.settings["use_distance_modification"].GetBool():
```

Comment:
There is no option to modify the distances inside the MLSConstraintProcess (in a avoiding small cuts sense).
There is the option to avoid zero distances in the MLSConstraintProcess, maybe that's what you mean? That could be also done in the solver instead. Then one more process than necessary would be called for the case that the user wants to use MLSConstraints for avoiding small cuts.

Basically ,the user can't decide whether zero distances are avoided. This is done anyway. Either in the MLSConstraintProcess or in the solver by calling the DistanceModificationProcess in the long run.

The user can decide whether he wants to use the MLSConstraintProcess to avoid small cuts or DistanceModification (tol=0.001) or nothing. If both options are set to true, the MLSConstraintProcess will be used.